OWML.Patcher/VrPatcher.cs

```diff
@@ -8,6 +8,7 @@ public class VRPatcher
         private readonly IOwmlConfig _owmlConfig;
         private readonly IModConsole _writer;
```

Comment:
No longer needed

Reply:
_owmlConfig seems to be used for .dll copying